src/main/resources/config/liquibase/master.xml

```diff
@@ -41,6 +41,9 @@
     <include file="classpath:config/liquibase/changelog/20241120213100_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20241114122713_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20241119191919_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20241125000900_changelog.xml" relativeToChangelogFile="false"/>
```

Comment:
two of these changesets are duplicated causing a startup issue (locally). removing `20241125000900_changelog` and `20241206114231_changelog `fixed the issue.